packages/components/src/components/radio-group/radio-group.stories.ts

```diff
@@ -41,280 +48,97 @@ export default {
  */
 
 export const Default = {
+  name: 'Default',
   render: (args: any) => {
     return generateTemplate({ args });
   }
 };
 
 /**
- * The sd-radio in all possible combinations of `orientation` and `size`.
- */
-
-export const Orientation = {
-  parameters: { controls: { exclude: ['orientation', 'size', 'default'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'orientation' },
-        y: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
-};
-
-/**
- * Use the disabled attribute to disable a input radio. Clicks will be suppressed until the disabled state is removed
- */
-
-export const Disabled = {
-  name: 'Disabled x Size',
-  parameters: { controls: { exclude: ['size', 'default'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: [
-          {
-            type: 'slot',
-            name: 'default',
-            title: 'disabled',
-            values: [
-              {
-                value:
-                  '<sd-radio value="1" disabled>Option 1</sd-radio><sd-radio value="2">Option 2</sd-radio><sd-radio value="3">Option 3</sd-radio>',
-                title: 'true'
-              },
-              {
-                value:
-                  '<sd-radio value="1">Option 1</sd-radio><sd-radio value="2">Option 2</sd-radio><sd-radio value="3">Option 3</sd-radio>',
-                title: 'false'
-              }
-            ]
-          }
-        ],
-        y: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
-};
-
-/**
- * Use the `invalid` attribute to mark the element is not in a format or a value the application will accept.
+ * Use the `size` attribute to set the size of the radio buttons. The default is `lg`. In tight spaces, consider using `sm`.
```

Comment:
Use the `size` attribute to change the size.

Reply:
Updated